src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml

```diff
@@ -1662,79 +1653,11 @@ let audit_type_shapes : Command.t =
          Core.printf "good shapes:\n\t%d\nbad shapes:\n\t%d\n%!" !good !bad ;
          if !bad > 0 then Core.exit 1 ) )
 
-[%%if force_updates]
```

Comment:
`force_updates` is set to `false` in every profile, so this code is dead. It no longer compiled, rather than update it I just deleted it.